osquery/config/config.cpp

```diff
@@ -530,6 +541,54 @@ void stripConfigComments(std::string& json) {
   json = sink;
 }
 
+std::map<std::string, std::string> Config::restoreConfigBackup() {
```

Comment:
Would you like to create an type alias for the `std::map<std::string, std::string>` and use it everywhere?

Reply:
A bit longer lines sometimes help, to not have to search declaration of those types. 

This type is used in a lot of places(not just in my newly created functions, as they get this container from the outside), if you think it helps, maybe we can do it another PR?